src/Navbar/Navbar.js

```diff
@@ -1,14 +1,23 @@
 import './Navbar.scss'
 
-function Navbar() {
-  return (
-    <nav className="navbar navbar-expand border-bottom shadow-sm px-4 justify-content-md-start justify-content-center">
-        <a href="#" className="navbar-brand fw-bold m-0">
-          <span className="brand-accent">Food</span>
-          <span>Delivery</span>
-        </a>
-    </nav>
-  )
+function Navbar(props) {
+    function handleChangeRestaurant() {
+        props.setDisplayRestaurantsOrMenu('show-restaurants')
+    }
+
+    return (
+        <nav className="navbar border-bottom shadow-sm px-4 flex-column flex-md-row">
+            <div>
+                <a href="#" className="navbar-brand fw-bold m-0">
```

Comment:
now that you have functionality to switch between the different "pages", clicking the logo should bring the user back to the page with all the restaurants, rather than doing nothing